### ISSUE TITLE
Did latest update break GUI's in free DR?

### ISSUE THREAD

**Issue by YourAverageMo** [YourAverageMo/auto-silence-cut#3]
From my understanding in the most recent DR update 19.1 they revoked GUI's in 3rd party scripts for free users. Meaning, in auto-silence-cut for example, unless you are on the studio (paid) version of DR the GUI thats coded into the script will not show preventing you from making any changes to the settings of the script.

Is this true? Would love to know if anyone has upgraded to the latest version.

**Comment by randyrektor** [comment]
Yes DR 19.1 removed GUI and third party scripts from the free version of DaVinci Resolve. Now the studio version is required. I couldn't find it in the release notes, but there are a few [Reddit threads](https://www.reddit.com/r/davinciresolve/comments/1gpo20i/191_broken_scripts_megathread/#:~:text=Service%20Notice%3A%20BMD%20has%20decided,Resolve%20Studio%2FFusion%20Studio%20v19.)

**Comment by YourAverageMo** [comment]
thank you randy for letting me know. thats unfortunate... im looking into other methods GUI so free users like me can still user it :D
